index.html

```diff
@@ -1,14 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>LAMBDA</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
+  <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>LAMBDA - healthy food restaurant</title>
 </head>
+
 <body>
-    <h1>LAMBDA restaurant</h1>
-    
+  <div class="wrapper">
```

Comment:
The structure is expected to be something like this
```html
<div class="main-wrapper">
  <main>
    <section class="section-promo">
      <header class="promo-header">
         <img />
         <nav>
           <ul>...</ul>
         </nav>
         <ul>...</ul>
       </header>
      <div class="promo-body">...</div>
    </section>
  </main>
</div>
```

Reply:
Fixed

Reply:
Not really. You put `<div class="logo-menu-bar">` which is redundant

Reply:
Corrected. Not sure I went the right way with regards to spacing between the logo and the menu.

Reply:
Use `flex-box`